examples/jsm/loaders/3DMLoader.js

```diff
@@ -181,26 +182,13 @@ class Rhino3dmLoader extends Loader {
 
 	_compareMaterials( material ) {
 
-		const mat = {};
-		mat.name = material.name;
-		mat.color = {};
-		mat.color.r = material.color.r;
-		mat.color.g = material.color.g;
-		mat.color.b = material.color.b;
-		mat.type = material.type;
+		const json = JSON.stringify( material );
```

Comment:
To get this PR merged, can you please revert the changes related to how materials are compared? Without the feedback of the original dev, I don't like to change this section.

Reply:
Done.